admiral/pkg/clusters/serviceentry.go

```diff
@@ -1605,6 +1610,16 @@ func AddServiceEntriesWithDrWorker(
 	}
 }
 
+func validateLocalityInServiceEntry(entry *v1alpha3.ServiceEntry) bool {
```

Comment:
Can this also go in quartermaster? May be jira in backlog would be good for now.